great_expectations_cloud/logging_cfg.py

```diff
@@ -54,5 +55,27 @@ def configure_logger(log_level: LogLevel, log_cfg_file: pathlib.Path | None) ->
         logging.config.dictConfig(dict_config)
         LOGGER.info(f"Configured logging from file {log_cfg_file}")
     else:
-        logging.basicConfig(level=log_level.numeric_level)
-        LOGGER.info(f"Configured logging at level {log_level}")
+        logDirectory = pathlib.Path("logs")
+        if not logDirectory.exists():
+            pathlib.Path(logDirectory).mkdir()
+
+        logger = logging.getLogger()
+        formatter = logging.Formatter(
+            "%(asctime)s | %(name)s | line: %(lineno)d | %(levelname)s: %(message)s"
+        )
+        logger.setLevel(logging.DEBUG)
```

Comment:
we have a `--log-level` flag that should be honored

Reply:
So apparently the root logger needs to be set up with a log level that is equal to or lower than the `FileHandler` and `StreamHandler`.  So I made the root logger use the lowest option.

Then the `StreamHandler` respects the user-provided `--log-level` (or defaults to `logging.WARN` if the user didn't provide the a `--log-level`.

I was assuming that we want the `FileHandler` to show the most verbose logging level possible, which is why I set it to `logging.DEBUG`.  If it defaults to just `logging.WARN`, will it give us enough info to debug issues? If so, I can update.

Reply:
Actually after more testing, the root logger `setLevel` is actually needed. I thought it was working without, but after more testing I realized that without setting the level of the root logger, it blocks DEBUG and INFO logs to the file. So I added it back and added a comment explaining why.